circuits/src/zk_circuits/valid_commitments.rs

```diff
@@ -341,7 +341,7 @@ where
     type Witness = ValidCommitmentsWitness<MAX_BALANCES, MAX_ORDERS, MAX_FEES>;
 
     fn name() -> String {
-        "Valid Commitments".to_string()
+        format!("Valid Commitments ({MAX_BALANCES}, {MAX_ORDERS}, {MAX_FEES})")
```

Comment:
i changed the names to include sizing constants so that the layout cache has different entries for different sizes -- this is necessary during tests